<launch>
    <node pkg="musi_care" type="sound_player" name="sound_player" output="screen"/>
    <node pkg="musi_care" type="qt_command_node" name="qt_command_node" output="screen"/>
    <node pkg="musi_care" type="guess_the_mood_game" name="guess_the_mood_game" output="screen"/>
</launch>
